app/views/provider_interface/start_page/show.html.erb

```diff
@@ -8,11 +8,12 @@
         <div class="app-masthead__description">
           <p class="govuk-body-l">Try our new service to see how easy it is to use. It’s free for providers and candidates.</p>
         </div>
-        <%= govuk_start_now_button(
-              text: 'Sign in',
-              href: provider_interface_sign_in_path,
-              classes: 'govuk-!-margin-bottom-0 app-button--inverted',
-            ) %>
+        <%= govuk_start_button(
+          text: 'Get started',
+          href: provider_interface_sign_in_path,
+          href: @pilot_enrolment_form_url,
+          classes: 'govuk-!-margin-bottom-0 app-button--inverted',
+        ) %>
```

Comment:
button text to change to `Sign in`, second `href` should be removed and `@pilot_enrolment_form_url` variable to be deleted.